pyracing/client.py

```diff
@@ -48,8 +42,9 @@ def __init__(self, username: str, password: str, log=default_logger()):
         self.password = password
         self.session = httpx.AsyncClient()
         self.log = log
+        self.__initial_auth = False
 
-    async def authenticate(self):
+    async def _authenticate(self):
```

Comment:
while this makes it private, it does not prevent it from being accessed. Private classes in Python are not made inaccessible, it's just a suggestion.